src/test/resources/org/elm/lang/core/parser/fixtures/partial/FieldAccessors.txt

```diff
@@ -8,57 +8,77 @@ Elm File
     PsiWhiteSpace(' ')
     PsiElement(EQ)('=')
     PsiWhiteSpace(' ')
+    ElmValueExpr(VALUE_EXPR)
+      ElmValueQID(VALUE_QID)
+        PsiElement(LOWER_CASE_IDENTIFIER)('x')
+  PsiErrorElement:<expr> or OPERATOR_IDENTIFIER expected, got '.'
+    PsiElement(DOT)('.')
+  PsiElement(VIRTUAL_END_DECL)('\n')
+  ElmValueDeclaration(VALUE_DECLARATION)
+    ElmFunctionDeclarationLeft(FUNCTION_DECLARATION_LEFT)
+      PsiElement(LOWER_CASE_IDENTIFIER)('f2')
+      PsiWhiteSpace(' ')
+      ElmLowerPattern(LOWER_PATTERN)
+        PsiElement(LOWER_CASE_IDENTIFIER)('x')
+    PsiWhiteSpace(' ')
+    PsiElement(EQ)('=')
+    PsiWhiteSpace(' ')
     ElmFieldAccessExpr(FIELD_ACCESS_EXPR)
       ElmValueExpr(VALUE_EXPR)
         ElmValueQID(VALUE_QID)
           PsiElement(LOWER_CASE_IDENTIFIER)('x')
       PsiElement(DOT)('.')
-      PsiErrorElement:<expr> or LOWER_CASE_IDENTIFIER expected
-        <empty list>
+      PsiElement(LOWER_CASE_IDENTIFIER)('bar')
+  PsiElement(DOT)('.')
+  PsiErrorElement:LOWER_CASE_IDENTIFIER expected
+    <empty list>
   PsiElement(VIRTUAL_END_DECL)('\n')
   ElmValueDeclaration(VALUE_DECLARATION)
     ElmFunctionDeclarationLeft(FUNCTION_DECLARATION_LEFT)
-      PsiElement(LOWER_CASE_IDENTIFIER)('f2')
+      PsiElement(LOWER_CASE_IDENTIFIER)('f3')
       PsiWhiteSpace(' ')
       ElmLowerPattern(LOWER_PATTERN)
         PsiElement(LOWER_CASE_IDENTIFIER)('x')
     PsiWhiteSpace(' ')
     PsiElement(EQ)('=')
     PsiWhiteSpace(' ')
-    ElmFieldAccessExpr(FIELD_ACCESS_EXPR)
-      ElmFieldAccessExpr(FIELD_ACCESS_EXPR)
-        ElmValueExpr(VALUE_EXPR)
-          ElmValueQID(VALUE_QID)
-            PsiElement(LOWER_CASE_IDENTIFIER)('x')
+    ElmValueExpr(VALUE_EXPR)
+      ElmValueQID(VALUE_QID)
+        PsiElement(UPPER_CASE_IDENTIFIER)('Foo')
         PsiElement(DOT)('.')
         PsiElement(LOWER_CASE_IDENTIFIER)('bar')
-      PsiElement(DOT)('.')
-      PsiErrorElement:<expr> or LOWER_CASE_IDENTIFIER expected
-        <empty list>
+  PsiElement(DOT)('.')
+  PsiErrorElement:LOWER_CASE_IDENTIFIER expected
+    <empty list>
   PsiElement(VIRTUAL_END_DECL)('\n')
+  PsiWhiteSpace('\n')
   ElmValueDeclaration(VALUE_DECLARATION)
     ElmFunctionDeclarationLeft(FUNCTION_DECLARATION_LEFT)
-      PsiElement(LOWER_CASE_IDENTIFIER)('f3')
+      PsiElement(LOWER_CASE_IDENTIFIER)('p1')
       PsiWhiteSpace(' ')
       ElmLowerPattern(LOWER_PATTERN)
         PsiElement(LOWER_CASE_IDENTIFIER)('x')
     PsiWhiteSpace(' ')
     PsiElement(EQ)('=')
     PsiWhiteSpace(' ')
-    ElmFieldAccessExpr(FIELD_ACCESS_EXPR)
-      ElmValueExpr(VALUE_EXPR)
-        ElmValueQID(VALUE_QID)
-          PsiElement(UPPER_CASE_IDENTIFIER)('Foo')
-          PsiElement(DOT)('.')
-          PsiElement(LOWER_CASE_IDENTIFIER)('bar')
-      PsiElement(DOT)('.')
-      PsiErrorElement:<expr> or LOWER_CASE_IDENTIFIER expected
-        <empty list>
+    ElmParenthesizedExpr(PARENTHESIZED_EXPR)
```

Comment:
GitHub isn't diffing this well, but pretty much only the error elements changed here.